blurb/blurb.py

```diff
@@ -751,10 +751,22 @@ def test(*args):
 
 
 def find_editor():
+    # Try environment variables. Check more specific settings first so
+    # they can override less specific values.
     for var in 'GIT_EDITOR', 'EDITOR':
         editor = os.environ.get(var)
         if editor is not None:
             return editor
+
+    # Try git configuration - this checks both project configuration
+    # (.git/config) and global configuration (~/.gitconfig).
+    editor = git_core_editor()
+    if editor:
+        return editor
```

Comment:
This should probably also be checked for existence, since on Windows it may give something inaccessible (something like `"vim"`, which Git itself interprets through its own synthetic UNIX-ish environment).  Basically, just make `git config --get core.editor` the first of the list of fallbacks if it gives anything at all.